src/components/AdvancedOptions.tsx

```diff
@@ -128,12 +129,14 @@ export default function AdvancedOptions(): ReactElement {
                 </ToggleButtonGroup>
                 <TextField
                   value={slippageCustom?.valueRaw}
+                  size="small"
+                  label=""
```

Comment:
You are right. Because before updating `TextField`, there is some hight difference between TextField with label and TextField without label, so I used "" label before, but because it was fixed, so we don't need this label.
Thank you for pointing this.